dist/angular-selector.js

```diff
@@ -375,9 +383,10 @@
 					if (!scope.multiple) scope.selectedValues = (scope.options || []).filter(function (option) { return scope.optionEquals(option); }).slice(0, 1);
 					else
 						scope.selectedValues = (scope.value || []).map(function (value) {
-							return $filter('filter')(scope.options, function (option) {
+							var inList = $filter('filter')(scope.options, function (option) {
 								return scope.optionEquals(option, value);
 							})[0];
+							return scope.createCustom ? value : inList;
```

Comment:
This allows the custom tags to be loaded back into the list. Otherwise only tags that exist in the options will be allowed through